ui/tests/acceptance/secrets/backend/database/secret-test.js

```diff
@@ -547,7 +547,7 @@ module('Acceptance | secrets/database/*', function (hooks) {
       .doesNotExist('Roles card does not exist on overview w/ policy');
     assert.dom('.overview-card h2').hasText('1', 'Lists the correct number of connections');
     // confirm get credentials card is an option to select. Regression bug.
-    await typeIn('.ember-text-field', 'blah');
+    await typeIn('[data-test-input-search="search-input-role"]', 'blah');
```

Comment:
A note: We have been slowly adding generic selectors to our `tests/helpers/general-selectors.ts` file, where this specific selector now exists. This file however, is old enough we don't import it. In the future if you run into a selector change, check if the test file imports `import { GENERAL } from 'vault/tests/helpers/general-selectors';`. If it does then you could do something like `GENERAL.inputSearch('search-input-role')`. Again, not something to change here, but more a note for the future.

Reply:
Duly noted :) (as in TIL)

I've added a commit here: https://github.com/hashicorp/vault/pull/29949/commits/5885bc0843eaaf6402528a1d50c172d539f5a881